ClosedXML/Word/XLDocStyle.cs

```diff
@@ -0,0 +1,62 @@
+﻿using DocumentFormat.OpenXml.Packaging;
+using DocumentFormat.OpenXml.Wordprocessing;
+
+namespace ClosedXML.Word
+{
+    public class XLDocStyle : IXLDocStyle
```

Comment:
In general, try not to abbreviate, i.e. use `XLDocumentStyle` instead of `XLDocStyle`.

Reply:
Okay, will do.